implementations/rust/ockam/ockam_command/src/cluster/inlet.rs

```diff
@@ -28,13 +29,8 @@ before_help = docs::before_help(PREVIEW_TAG),
 after_long_help = docs::after_help(AFTER_LONG_HELP)
 )]
 pub struct InletCommand {
-    /// The Cluster that hosts the Zone.
-    #[arg(long)]
-    pub cluster: Option<String>,
```

Comment:
we *cant* infer the cluster name in the general case.
Being it  optional is ok.  For simple examples where everything is run by the same person on the same machine, letting it unspecified and retrieving it from orchestrator is fine (it is expected the user is enrolled, as is the one creating the zone, etc).

But breaks as soon as one tries to create the inlet in some machine that is *not* enrolled with ockam ai,   or if one want to create the inlet pointing to someone else cluster,  with a ticket that that other user gives us.

Reply:
Yes, my bad... it only makes sense to infer the cluster from the "cluster creator" perspective right?

Reply:
Fixed!